walk-in-interview/src/main/webapp/log-in/applicant/script.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * This file is specific to log-in/applicant/index.html. 
+ * It renders the fields on the page dynamically
+ */
+
+// TODO(issue/21): get the language from the browser
+const CurrentLocale = 'en';
+
+/**
+ * Import statements are static so its parameters cannot be dynamic.
+ * TODO(issue/22): figure out how to use dynamic imports
+ */
+import {AppStrings} from '../../strings.en.js';
+
+const COMMONG_STRINGS = AppStrings['log-in'];
+const STRINGS = AppStrings['applicant-log-in'];
+const HOMEPAGE_PATH = '../index.html';
+
+window.onload = () => {
+  renderPageElements();
+}
+
+/** Adds all the text to the fields on this page. */
+function renderPageElements() {
+  const accountElement = document.getElementById('account');
+
+  const accountLabelElement = accountElement.children[0];
+  accountLabelElement.innerText = STRINGS['account'];
+
+  const accountInputElement = accountElement.children[1];
+  accountInputElement.setAttribute('name', 'account-input');
+  accountInputElement.setAttribute('type', 'text');
+  // TODO(issue/78): send the input to the firebase auth related stuff
```

Comment:
im not sure i understand this TODO. I understand why it would be in the onClick method below but not why it's in the renderPageelements (same for the other one in that same method).